packages/flutter/lib/src/widgets/focus.dart

```diff
@@ -44,8 +44,8 @@ class _FocusScope extends InheritedWidget {
     focusedScope = focusState._focusedScope == _noFocusedScope ? null : focusState._focusedScope;
   }
 
-  void _setFocusedScopeIfUnset(GlobalKey key) {
-    focusState._setFocusedScopeIfUnset(key);
+  void _setFocusedScope(GlobalKey key) {
```

Comment:
this function is now unused as far as I can tell, and focusedScope should now be final.